controllers/ocirepository_controller_test.go

```diff
@@ -808,6 +1412,22 @@ func createPodinfoImageFromTar(tarFileName, tag string, imageServer *httptest.Se
 	}, nil
 }
 
+func pushMultiplePodinfoImage(serverURL string, versions []string) (map[string]podinfoImage, error) {
```

Comment:
Nit:

```suggestion
func pushMultiplePodinfoImages(serverURL string, versions ...string) (map[string]podinfoImage, error) {
```

would result in a nicer API.